<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>emp</class>
 <widget class="QDialog" name="emp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>130</y>
     <width>801</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string>ajouter</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>110</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>cin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>adresse</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>170</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>prenom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>226</y>
       <width>56</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>mail</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nom</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_mail">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>230</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_ad">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_prenom">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>170</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_nom">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_cin">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>110</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonajouter">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>450</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { background-color: #f0f0f0; border: 1px solid #d0d0d0; }
QPushButton:hover { background-color: #e0e0e0; }
QPushButton:pressed { background-color: #d0d0d0; }</string>
     </property>
     <property name="text">
      <string>ajouter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>821</width>
       <height>551</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../Capture.JPG</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>270</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>absence</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>270</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>310</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Mot de passe</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="motdepasse">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>310</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>350</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirmer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="motdepasse_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>350</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QComboBox" name="Role">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>410</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Visiteur</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Admin</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Pr√©sentateur</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>410</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Role</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonajouter_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>380</y>
       <width>96</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { background-color: #f0f0f0; border: 1px solid #d0d0d0; }
QPushButton:hover { background-color: #e0e0e0; }
QPushButton:pressed { background-color: #d0d0d0; }</string>
     </property>
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="RFID">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>380</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>380</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>RFID</string>
     </property>
    </widget>
    <zorder>label_12</zorder>
    <zorder>label</zorder>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
    <zorder>lineEdit_mail</zorder>
    <zorder>lineEdit_ad</zorder>
    <zorder>lineEdit_prenom</zorder>
    <zorder>lineEdit_nom</zorder>
    <zorder>lineEdit_cin</zorder>
    <zorder>pushButtonajouter</zorder>
    <zorder>label_18</zorder>
    <zorder>lineEdit</zorder>
    <zorder>label_20</zorder>
    <zorder>motdepasse</zorder>
    <zorder>label_21</zorder>
    <zorder>motdepasse_2</zorder>
    <zorder>Role</zorder>
    <zorder>label_22</zorder>
    <zorder>pushButtonajouter_2</zorder>
    <zorder>RFID</zorder>
    <zorder>label_23</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <property name="styleSheet">
     <string notr="true">QWidget { background-color: #f0f0f0; border: 1px solid #d0d0d0; }</string>
    </property>
    <attribute name="title">
     <string>afficher</string>
    </attribute>
    <widget class="QTableView" name="tableView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>471</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>761</width>
       <height>531</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../Capture.JPG</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>190</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_3">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>100</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>CIN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NOM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PRENOM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ADRESSE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MAIL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ABSENCE</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton1">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>190</y>
       <width>96</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { background-color: #f0f0f0; border: 1px solid #d0d0d0; }
QPushButton:hover { background-color: #e0e0e0; }
QPushButton:pressed { background-color: #d0d0d0; }</string>
     </property>
     <property name="text">
      <string>annuler</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>240</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { background-color: #f0f0f0; border: 1px solid #d0d0d0; }
QPushButton:hover { background-color: #e0e0e0; }
QPushButton:pressed { background-color: #d0d0d0; }</string>
     </property>
     <property name="text">
      <string>export PDF</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>290</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { background-color: #f0f0f0; border: 1px solid #d0d0d0; }
QPushButton:hover { background-color: #e0e0e0; }
QPushButton:pressed { background-color: #d0d0d0; }</string>
     </property>
     <property name="text">
      <string>statistique</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>350</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { background-color: #f0f0f0; border: 1px solid #d0d0d0; }
QPushButton:hover { background-color: #e0e0e0; }
QPushButton:pressed { background-color: #d0d0d0; }</string>
     </property>
     <property name="text">
      <string>tri</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>390</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton { background-color: #f0f0f0; border: 1px solid #d0d0d0; }
QPushButton:hover { background-color: #e0e0e0; }
QPushButton:pressed { background-color: #d0d0d0; }</string>
     </property>
     <property name="text">
      <string>annuler</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>360</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>selon</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_4">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>360</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>CIN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NOM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PRENOM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ADRESSE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MAIL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ABSENCE</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>150</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <zorder>label_14</zorder>
    <zorder>tableView</zorder>
    <zorder>label_16</zorder>
    <zorder>comboBox_3</zorder>
    <zorder>pushButton1</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>pushButton_4</zorder>
    <zorder>pushButton_5</zorder>
    <zorder>label_17</zorder>
    <zorder>comboBox_4</zorder>
    <zorder>lineEdit_2</zorder>
   </widget>
   <widget class="QWidget" name="tab_3">
    <property name="styleSheet">
     <string notr="true">QWidget { background-color: #f0f0f0; border: 1px solid #d0d0d0; }</string>
    </property>
    <attribute name="title">
     <string>modifier</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>90</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>cin</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>mail</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>160</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>prenom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nom</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>210</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>adresse</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditNN">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>120</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_P">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_A">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_OO">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>250</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonMODIFIER">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>190</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>modifier</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>781</width>
       <height>531</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../Capture.JPG</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>300</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>absence</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>300</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <zorder>label_13</zorder>
    <zorder>label_7</zorder>
    <zorder>label_8</zorder>
    <zorder>label_9</zorder>
    <zorder>label_10</zorder>
    <zorder>label_11</zorder>
    <zorder>comboBox_2</zorder>
    <zorder>lineEditNN</zorder>
    <zorder>lineEdit_P</zorder>
    <zorder>lineEdit_A</zorder>
    <zorder>lineEdit_OO</zorder>
    <zorder>pushButtonMODIFIER</zorder>
    <zorder>label_19</zorder>
    <zorder>lineEdit_3</zorder>
   </widget>
   <widget class="QWidget" name="tab_4">
    <property name="styleSheet">
     <string notr="true">QWidget { background-color: #f0f0f0; border: 1px solid #d0d0d0; }</string>
    </property>
    <attribute name="title">
     <string>supprimer</string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>56</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>CIN</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>110</y>
       <width>73</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="supprimer">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>140</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>5</y>
       <width>771</width>
       <height>521</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../Capture.JPG</pixmap>
     </property>
    </widget>
    <zorder>label_15</zorder>
    <zorder>label_6</zorder>
    <zorder>comboBox</zorder>
    <zorder>supprimer</zorder>
   </widget>
  </widget>
  <widget class="QLabel" name="Connected">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>70</y>
     <width>321</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 7pt &quot;MS Serif&quot;;
color: rgb(255, 85, 0)</string>
   </property>
   <property name="text">
    <string>Connect√© en tant que: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
